kvstore/kvstore_server.h

```diff
@@ -9,14 +13,15 @@ using kvstore::PutRequest;
 using kvstore::PutReply;
 using kvstore::GetRequest;
 using kvstore::GetReply;
+using kvstore::RemoveRequest;
+using kvstore::RemoveReply;
 
 class KvstoreServiceImpl final : public KeyValueStore::Service {
 public:
   Status put(ServerContext* context, const PutRequest* request, PutReply* reply){}
   Status get(ServerContext* context, const GetRequest* request, GetReply* reply){}
   Status remove(ServerContext* context, const RemoveRequest* request, RemoveReply* reply){}
```

Comment:
you don't need the {}, only a semi-colon since this is the header and all you're doing is declaring it and you've already written the definitions in the .cc file

Reply:
Deleted